gtsam/sfm/MFAS.cpp

```diff
@@ -7,6 +7,8 @@
 
 #include <gtsam/sfm/MFAS.h>
 
+#include <boost/shared_ptr.hpp>
```

Comment:
Is shared_ptr being used right now? I see it has been removed from one instance.